astroquery/linelists/cdms/core.py

```diff
@@ -0,0 +1,314 @@
+            Identifiers of the molecules to search for. If this parameter
+            is not provided the search will match any species. Default is 'All'.
+
+        temperature_for_intensity : float
+            The temperature to use when computing the intensity Smu^2.  Set
+            to 300 by default for compatibility with JPL and the native
+            catalog format, which defaults to 300.
+            ** If temperature is set to zero, the return value in this column
+            will be the Einstein A value **
+
+        flags : int, optional
+            Regular expression flags. Default is set to 0
+
+        parse_name_locally : bool, optional
+            When set to True it allows the method to parse through catdir.cat
+            (see `get_species_table`) in order to match the regex inputted in
+            the molecule parameter and request the corresponding tags of the
+            matches instead. Default is set to False
+
+        get_query_payload : bool, optional
+            When set to `True` the method should return the HTTP request
+            parameters as a dict. Default value is set to False
+
+        cache : bool
+            Cache the request and, for repeat identical requests, reuse the
+            cache?
+
+        Returns
+        -------
+        response : `requests.Response`
+            The HTTP response returned from the service.
+
+        Examples
+        --------
+        >>> table = CDMS.query_lines(min_frequency=100*u.GHz,
+        ...                          max_frequency=110*u.GHz,
+        ...                          min_strength=-500,
+        ...                          molecule="018505 H2O+") # doctest: +REMOTE_DATA
+        >>> print(table) # doctest: +SKIP
+            FREQ     ERR   LGINT   DR   ELO    GUP  TAG  QNFMT  Ju  Ku  vu  Jl  Kl  vl      F      name
+            MHz      MHz  MHz nm2      1 / cm
+        ----------- ----- ------- --- -------- --- ----- ----- --- --- --- --- --- --- ----------- ----
+        103614.4941 2.237 -4.1826   3 202.8941   8 18505  2356   4   1   4   4   0   4 3 2 1 3 0 3 H2O+
+        107814.8763 148.6 -5.4438   3 878.1191  12 18505  2356   6   5   1   7   1   6 7 4 4 8 1 7 H2O+
+        107822.3481 148.6 -5.3846   3 878.1178  14 18505  2356   6   5   1   7   1   7 7 4 4 8 1 8 H2O+
+        107830.1216 148.6 -5.3256   3 878.1164  16 18505  2356   6   5   1   7   1   8 7 4 4 8 1 9 H2O+
+        """
+        # first initialize the dictionary of HTTP request parameters
+        payload = dict()
+
+        if min_frequency is not None and max_frequency is not None:
+            # allow setting payload without having *ANY* valid frequencies set
+            min_frequency = min_frequency.to(u.GHz, u.spectral())
+            max_frequency = max_frequency.to(u.GHz, u.spectral())
+            if min_frequency > max_frequency:
+                raise InvalidQueryError("min_frequency must be less than max_frequency")
```

Comment:
ValueError here was more appropriate as at this point there is no query. Sorry about that.

Reply:
I think this is still more informative to users